src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs

```diff
@@ -673,7 +673,11 @@ protected override void OnPointerWheelChanged(PointerWheelEventArgs e)
                 {
                     delta = new Vector(delta.Y, delta.X);
                 }
-                
+                else
+                {
+                    delta = AdjustDeltaForFlowDirection(delta, FlowDirection);
```

Comment:
This should be done unconditionally, not in an `else` block. Otherwise, shift + scroll wheel is still reversed.

Reply:
I am not sure about this. I think shift+scroll should remained the original behavior, as scrolling up means to show later content. Maybe ask some native Arabic speakers' opinion like @Mop1398

Reply:
Afaict, at least on browsers, shift+wheel up scrolls left even with `direction: rtl`.
But I'm no expert here, let's wait for a native speaker's input.

Reply:
Note: I've searched for, but couldn't find any other behavior than Shift+Left scrolling Up and Shift+Right scrolling Down, whatever the text direction. 

I don't want to hold this PR, as the original issue is fixed. I'm going to merge it. We can revisit the Shift+Left/Right scrolling in another issue if needed.